lib/resources/usVerifications.js

```diff
@@ -8,8 +8,13 @@ class USVerifications extends ResourceBase {
     super('us_verifications', config);
   }
 
-  verify (params, callback) {
-    return this._transmit('POST', null, null, params, callback);
+  verify (params, qs, callback) {
+    if (typeof qs === 'function') {
+      callback = qs;
+      qs = {};
```

Comment:
should this say `qs = null` because `{}` will evaluate to `true`? and `null` was passed in before for `qs`

Reply:
I'm not sure if `null` was a good default to begin with given that it's taking an object according to the [docs](https://github.com/request/request#requestoptions-callback).